docs/conf.py

```diff
@@ -17,32 +17,22 @@
 
 # General information about the project.
 
-fullversion = os.environ["BUILD_VERSION"]
-release = fullversion
-
-version = fullversion
-
-m = re.match(r"^(v\d+\.\d+\.\d+)(-rc\d+)", version)
-
-if m:
-    version = "".join(m.groups())
-
-release = version
+version = os.environ["ENVOY_GATEWAY_VERSION"]
+envoyVersion = os.environ["ENVOY_VERSION"]
+gatewayAPIVersion = os.environ["GATEWAYAPI_VERSION"]
 
 project = f'Envoy Gateway {version}'
 author = 'Envoy Gateway Project Authors'
 
-copyright = '2022 Envoy Gateway Project Authors | ' + fullversion
-
-envoyVersion = os.environ["ENVOY_VERSION"]
-gatewayAPIVersion = os.environ["GATEWAYAPI_VERSION"]
+copyright = '2022 Envoy Gateway Project Authors | ' + version
```

Comment:
Let's drop the year from the copyright.

xref: https://github.com/envoyproxy/gateway/pull/590/files